### ISSUE TITLE
Prefers Reduced Motion and Header Behavior

### ISSUE THREAD

**Issue by jodylecompte** [jodylecompte/jodylecompte.com#3]
Currently on scroll, the top menu bar does not have fixed positioning and scrolls back into view as soon as the user scrolls up.

Ideally, this could be placed behind a media query for prefers reduced motion so that this behavior is the default but in cases where preferred reduced motion is true, the header remains in a fixed position and is visible at all times.
